store/copr/mpp.go

```diff
@@ -341,13 +341,21 @@ func (m *mppIterator) cancelMppTasks() {
 	}
 
 	// send cancel cmd to all stores where tasks run
+	wg := new(sync.WaitGroup)
```

Comment:
Use this

https://github.com/pingcap/tidb/blob/41fb3e75a51df36100d6dd5083a9332e5a88dc7d/util/wait_group_wrapper.go#L22

Reply:
Done